src/sections/trade/sections/swap/SwapPage.tsx

```diff
@@ -50,6 +54,8 @@ export function SwapPage() {
   const { createTransaction } = useStore()
   const { stableCoinId } = useDisplayAssetStore()
 
+  const isEvm = isEvmAddress(account?.displayAddress ?? "")
```

Comment:
```suggestion
  const isEvm = isEvmAccount(account?.address)
```
I would use this check instead